From 7a3fe674f05dcd333541b5d86922d02d36134352 Mon Sep 17 00:00:00 2001
From: Wincent Colaiuta <win@wincent.com>
Date: Tue, 29 Jun 2010 19:09:10 +0200
Subject: [PATCH] docs: note that mail() accepts arbitrary headers

The documentation makes it sound like mail() only accepts a limited
set of headers in the headers hash, but it in fact accepts any
arbitrary headers, and there is a test ("can pass random headers in
as a hash to mail") for it in the test suite.

Signed-off-by: Wincent Colaiuta <win@wincent.com>
---
 actionmailer/lib/action_mailer/base.rb |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/actionmailer/lib/action_mailer/base.rb b/actionmailer/lib/action_mailer/base.rb
index 3b6e2c6..11fa978 100644
--- a/actionmailer/lib/action_mailer/base.rb
+++ b/actionmailer/lib/action_mailer/base.rb
@@ -535,7 +535,9 @@ module ActionMailer #:nodoc:
     #                 :reply_to => 'bounces@test.lindsaar.net'
     #  end
     #
-    # If you need other headers not listed above, use the <tt>headers['name'] = value</tt> method.
+    # If you need other headers not listed above, you can either pass them in
+    # as part of the headers hash or use the <tt>headers['name'] = value</tt>
+    # method.
     #
     # When a <tt>:return_path</tt> is specified as header, that value will be used as the 'envelope from'
     # address for the Mail message.  Setting this is useful when you want delivery notifications
-- 
1.7.1

